packages/components/src/Button/Button.css

```diff
@@ -73,39 +73,85 @@ a.button:active {
 
 /* Cancel is special because, by default, it's styled as a secondary button */
 
-.cancel {
-  --button--color-variation: var(--color-cancel);
-  --button--color-variation--hover: var(--color-cancel--hover);
+.cancel,
+.subtle {
+  --button--color-variation: var(--color-interactive--subtle);
+  --button--color-variation--hover: var(--color-interactive--subtle--hover);
 }
 
 /* Type */
 
-.primary:not(.cancel),
-.primary:not(.cancel):hover,
-.primary:not(.cancel):focus {
+.primary:not(.cancel, .subtle),
```

Comment:
I have opted to used the selector list argument here because

1. reduced number of total lines of CSS to read
2. doing it in separate lines appeared to cancel out whichever one was listed second